controller/instance_manager_controller.go

```diff
@@ -443,6 +550,34 @@ func (imc *InstanceManagerController) enqueueInstanceManagerPod(pod *v1.Pod) {
 	imc.enqueueInstanceManager(im)
 }
 
+func (imc *InstanceManagerController) enqueueKubernetesNode(n *v1.Node) {
+	node, err := imc.ds.GetNode(n.Name)
+	if err != nil {
+		if apierrors.IsNotFound(err) {
+			// there is no Longhorn node created for the Kubernetes
+			// node (e.g. controller/etcd node). Skip it
+			return
+		}
+		utilruntime.HandleError(fmt.Errorf("Couldn't get node %v: %v ", n.Name, err))
+		return
+	}
+
+	// We only enqueue engine instance manager for syncInstanceManagerPDB
+	ims, err := imc.ds.ListInstanceManagersByNode(node.Name, types.InstanceManagerTypeEngine)
+	if err != nil {
+		if apierrors.IsNotFound(err) {
+			logrus.Warnf("Can't find instance manager for node %v, may be deleted", node.Name)
+			return
+		}
+		utilruntime.HandleError(fmt.Errorf("couldn't get instance manager: %v", err))
+		return
+	}
+
+	for _, im := range ims {
+		imc.enqueueInstanceManager(im)
+	}
+}
```

Comment:
I noticed that the InstanceManager is slow to react when node changes status (schedulable/unschedulable).

Therefore, I added to the `kubeNodeInformer` so that InstanceManager react faster.